.github/workflows/destroy-environment.yml

```
@@ -0,0 +1,54 @@
+name: Destroy Environment
+run-name: Destroying environments starting with ${{ github.event.inputs.prefix }} by @${{ github.actor }}
+
+on:
+  # Ability to execute on demand
+  workflow_dispatch:
+    inputs:
+      ec-api-key:
+        type: string
+        description: "Elastic Cloud API KEY"
+        required: true
+      prefix:
+        type: string
+        description: "delete all cloud environments that start with {{prefix}}"
```

can we use `prefix` here?
```suggestion
        description: "Delete all environments starting with `prefix`"
```